base/docs/helpdb.jl

```diff
@@ -1335,11 +1335,11 @@ Get the value type of an associative collection type. Behaves similarly to `elty
 valtype
 
 doc"""
-    edit(file::AbstractString, [line])
+    edit(path::AbstractString, [line])
 
-Edit a file optionally providing a line number to edit at. Returns to the julia prompt when you quit the editor.
+Edit a file or directory optionally providing a line number to edit the file at. Returns to the julia prompt when you quit the editor.
 """
-edit(file::AbstractString, line=?)
+edit(path::AbstractString, line=?)
```

Comment:
What does the question mark do in this context? I see it's in a few other places in helpdb but I've never seen this syntax before, and it's difficult to search for...